sstables_loader.cc

```diff
@@ -487,10 +490,9 @@ future<> sstables_loader::download_task_impl::run() {
     sstables::sstable_open_config cfg {
         .load_bloom_filter = false,
     };
-    auto prefix = format("{}/{}", _cf, _snapshot_name);
-    llog.debug("Loading sstables from {}({}/{})", _endpoint, _bucket, prefix);
-    auto [ table_id, sstables_on_shards ] = co_await replica::distributed_loader::get_sstables_from_object_store(_loader.local()._db, _ks, _cf, _endpoint, _bucket, prefix, cfg);
-    llog.debug("Streaming sstables from {}({}/{})", _endpoint, _bucket, prefix);
+    llog.debug("Loading sstables from {}({}/{})", _endpoint, _bucket, _prefix);
+    auto [ table_id, sstables_on_shards ] = co_await replica::distributed_loader::get_sstables_from_object_store(_loader.local()._db, _ks, _cf, _sstables, _endpoint, _bucket, _prefix, cfg);
```

Comment:
is it possible just to `auto&& [ table_id, sstables_on_shards ]` and avoid possible copying